level1c4pps/vgac2pps_lib.py

```diff
@@ -28,6 +28,7 @@
 import os
 import time
 from satpy.scene import Scene
+from sbafs_ann.convert_vgac import convert_to_vgac_with_nn
```

Comment:
this module is missing

Reply:
I know what to do about this.

Reply:
Ok. Nina is on vacation today, but I talked to her the last week. She said she plans to add this package to PyPI, but it is not yet there. When it is there we need to add it as a dependency in the setup.py file in the same way as e.g. satpy is listed, but until this is fixed all tests will fail on github even if you can run it locally...